pages/domains/[name]/index.tsx

```diff
@@ -380,6 +400,14 @@ export function ManageNamePageContent({ name }: { name: string }) {
           }}
         />
       </Modal>
+      <Modal open={editResolverModalOpen} onDismiss={() => {}}>
+        <EditResolverModalContent
+          name={name}
+          closeModal={() => {
```

Comment:
I think that would be nice to pass the resolver address here as a variable and set the resolver address state inside the modal as the current resolver address

Reply:
@lgahdl Great catch! Done